swan-cern/files/swan_config_cern.py

```diff
@@ -42,6 +45,18 @@ async def get_swan_user_pod(self):
 
         return self.pod
 
+    def _check_tn_access(self):
+        """
+        Helper function to check if this SWAN deployment is exposed to the Technical Network
+        and this SWAN deployment is exposed to the Technical Network
```

Comment:
@rodrigo-sobral please check your commits before asking to review again. This comment has a repetition.